buildlib/azure-pipelines-pr.yml

```diff
@@ -12,7 +12,7 @@ resources:
       image: ucfconsort.azurecr.io/ucx/centos7:1
       endpoint: ucfconsort_registry
     - container: fedora
-      image: ucfconsort.azurecr.io/ucx/fedora:1
+      image: ucfconsort.azurecr.io/ucx/fedora:2
```

Comment:
@amaslenn what does this change mean?

Reply:
I've built a new docker image adding `java-1.8.0-openjdk-devel` package to satisfy dependencies of RPM build. This number is a Docker tag, similar to `git` tags.